### PR TITLE
chore(deps): bump lodash from 4.17.16 to 4.17.19

### PR DESCRIPTION
**Pull request by Crow-EH**
Hi ! :wave: 

Advisory : https://www.npmjs.com/advisories/1523

I don't know why renovate didn't update that one yet.

### CODE REVIEW

**Comment by charlierudolph** [comment]
I configured renovate to run once a month (to avoid too much noise)

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 * don't execute BeforeAll and AfterAll hooks when in dry-run
 * support correct case for `--retry-tag-filter` CLI argument
 
+#### Internals
+
+* Bump Lodash
```
Comment (charlierudolph) [inline CHANGELOG.md:26]: Don't think this really belongs in the changelog as not something I expect a user to care about.

Comment (Crow-EH) [reply CHANGELOG.md]: Done

**Comment by aslakhellesoy** [comment]
Hi @Crow-EH,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**